### PR TITLE
Automatically create a thread for fdmds messages

### PR DESCRIPTION
**Pull request by MineKing9534**
## Checklist

 - [x] Read [Contribution Guidelines](https://github.com/SlimeCloud/java-SlimeBot#-contributing) and [Style Guides](https://github.com/SlimeCloud/java-SlimeBot#-style-guide)
 - [x] Test the Code
 - [x] This PR is ready to review and merge

## Description
Automatically creates a thread for every fdmds message

### CODE REVIEW

File: src/main/java/com/slimebot/commands/FdmdsCommand.java [hunk 1]
```diff
@@ -92,7 +92,7 @@ public void handleFdmdsModal(ModalInteractionEvent event) {
 			embedBuilder.setFooter(event.getMessage().getEmbeds().get(0).getFooter().getText())
 					.addField("Frage:", question, false);
 		} else if (event.getModalId().contains("send")) {
-			embedBuilder.addField("Frage:", "Heute würde ich gerne von euch wissen, " + question.split(" ", 2)[0].toLowerCase() + " " + question.split(" ", 2)[1], false);
```
Comment (MatitoDev) [inline src/main/java/com/slimebot/commands/FdmdsCommand.java:95]: i think i added this because it's weird when there's a random capitalised word in the middle of a sentence just because the user started with a capital letter when submitting the question

why did you remove this?

**Review by MatitoDev — CHANGES_REQUESTED** [review]

File: src/main/java/com/slimebot/commands/FdmdsCommand.java (hunk 1 above)
Comment (MineKing9534) [reply src/main/java/com/slimebot/commands/FdmdsCommand.java:95]: because it breaks when the user sends a message with just one word. In practice this probably doesn't happen but it would throw an exception if some random troll does so. 
I think a better solution would be `question.substring(0, 1).toLowerCase() + question.substring(1)`

**Review by MatitoDev — APPROVED** [review]
